shared/team-building/container.tsx

```diff
@@ -566,7 +570,14 @@ const mergeProps = (
         ],
         title,
       }
-    : {}
+    : emptyObj
+
+  const onChangeService = (service: Types.ServiceIdWithContact) => {
+    ownProps.onChangeService(service)
+    if (!Types.isContactServiceId(service)) {
+      dispatchProps._search(ownProps.searchString, service)
```

Comment:
What happens when it is contact service id?

Reply:
The email/phone tabs don't share `searchString` with the rest of the tabs, so they'll start out empty and call `search` when they see a valid phone or email.